filebeat/docs/fields.asciidoc

```diff
@@ -97685,6 +97686,257 @@ alias to: source.ip
 
 --
 
+[[exported-fields-mysql]]
+== mysql fields
```

Comment:
You'll want to specify that this is for the x-pack MySQL enterprise module versus the [OSS MySQL module](https://github.com/elastic/beats/tree/master/filebeat/module/mysql). Something like `MySQL Enterprise Fields`?